README.md

```diff
@@ -1,3 +1,48 @@
 # Info Support Tech Survey
 
-## Introduction
+## Setup for local development
+
+### Setup database
+
+1. Make sure docker desktop or another tool of your liking is installed on your machine.
+2. Spin up a postgres database with the following command. You need to choose your own password and username:
+```bash
+docker run --name tech-survey -e "POSTGRES_USER=dummyusr" -e "POSTGRES_PASSWORD=dummypw" -e "POSTGRES_DB=tech-survey" -d -p 5432:5432 docker.io/postgres
+```
+3. Make sure to also use that username and password inside of the `.env` file:
+`DATABASE_URL="postgresql://jarne:dummyusr@dummypw/tech-survey`

```

Comment:
Generally, it's a good idea not to include your name in the (demo) source code.

Reply:
Whoops, that was not the intention, adjusted!